Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -5622,4 +5622,58 @@
         <WhichUnit Value="Source"/>
         <Value value="AP_PhoenixTaldarim"/>
     </CValidatorUnitType>
+    <CValidatorPlayerRequirement id="AP_HaveTyrannozorTyrantsProtection">
+        <Find value="1"/>
+        <Value value="AP_HaveTyrannozorTyrantsProtection"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorLocationEnumArea id="AP_TyrannozorBarrageOfSpikesEnemiesNearby">
+        <AreaArray Compare="GE" Count="1" Radius="2"/>
+        <SearchFilters value="Visible;Self,Player,Ally,Neutral,Air,Structure,RawResource,HarvestableResource,Missile,Stasis,Dead,Hidden,Invulnerable"/>
+    </CValidatorLocationEnumArea>
+    <CValidatorPlayerRequirement id="AP_HaveImpalingStrike">
+        <Find value="1"/>
+        <Value value="AP_HaveImpalingStrike"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorPlayerRequirement id="AP_HaveChitinousPlating">
+        <Find value="1"/>
+        <Value value="AP_HaveChitinousPlating"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorPlayerRequirement id="AP_HaveAnabolicSynthesis">
+        <Find value="1"/>
+        <Value value="AP_HaveAnabolicSynthesis"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorPlayerRequirement id="AP_HaveHealingAdaptation">
+        <Find value="1"/>
+        <Value value="AP_HaveHealingAdaptation"/>
+    </CValidatorPlayerRequirement>
+    <CValidatorPlayerCompareFoodAvailable id="AP_CheckTyrannozorMorphlingMergeSupply">
+        <ResultFailed value="NotEnoughFood"/>
+        <WhichPlayer Value="Caster"/>
+        <Compare value="LT"/>
+        <Value value="5"/>
+    </CValidatorPlayerCompareFoodAvailable>
+    <CValidatorPlayerCompareFoodAvailable id="AP_CheckTyrannozorMorphlingMergeSupplyResourceEfficiency">
+        <ResultFailed value="NotEnoughFood"/>
+        <WhichPlayer Value="Caster"/>
+        <Compare value="LT"/>
+        <Value value="3"/>
+    </CValidatorPlayerCompareFoodAvailable>

```

Comment:
Should be 2 supply (Tyrannozor is 6 supply if upgraded, morphlings are 2 supply each)
![Snímek obrazovky pořízený 2024-07-28 18-51-29](https://github.com/user-attachments/assets/e9b89a88-cfcc-4b8f-92b4-938cb8278a01)
I have enough supply available but still can't merge

Reply:
Fixed